content/get-started/using-github/keyboard-shortcuts.md

```diff
@@ -38,7 +38,7 @@ Below is a list of some of the available keyboard shortcuts.
 |<kbd>g</kbd> <kbd>b</kbd> | Go to the **Projects** tab. For more information, see "[About project boards](/articles/about-project-boards)."
 |<kbd>g</kbd> <kbd>w</kbd> | Go to the **Wiki** tab. For more information, see "[About wikis](/communities/documenting-your-project-with-wikis/about-wikis)."{% ifversion fpt %}
 |<kbd>g</kbd> <kbd>g</kbd> | Go to the **Discussions** tab. For more information, see "[About discussions](/discussions/collaborating-with-your-community-using-discussions/about-discussions)."{% endif %}
-
+|<kbd>.</kbd>| Open in github.dev editor
```

Comment:
This is a great suggestion, but I would suggest maybe putting it at the top of the "Source code editing" table, as you can open more than just repositories in the editor. I also updated the language to reflect the current naming conventions that we're using. 

```suggestion
|<kbd>.</kbd>| Opens a repository or pull request in web-based editor. For more information, see "[Web-based editor](/codespaces/developing-in-codespaces/web-based-editor)."
```